src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/sideeffects/CardEventHandler.java

```diff
@@ -1,10 +1,7 @@
 package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.sideeffects;
 
-import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Card;
-import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Match;
-import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Round;
-import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.repository.CardRepository;
-import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.repository.RoundRepository;
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.*;
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.repository.*;
 import javax.transaction.Transactional;
 import org.springframework.data.rest.core.annotation.HandleAfterSave;
 import org.springframework.data.rest.core.annotation.RepositoryEventHandler;
```

Comment:
```suggestion
import org.springframework.data.rest.core.annotation.RepositoryEventHandler;

@Component
```
Das aktiviert das auto wiring, und dann kommen die dependencies automatisch.
Wobei es scheint auch ohne zu funktionieren...?